### PR TITLE
Fix two e2e tests

### PR DESCRIPTION
**Pull request by cevich**
These tests are failing due to slightly wider coverage in #6784 but otherwise should be fixed anyway.

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL

File: test/e2e/exec_test.go [hunk 1]
```diff
@@ -286,7 +286,7 @@ var _ = Describe("Podman exec", func() {
 	It("podman exec preserves container groups with --user and --group-add", func() {
 		SkipIfRemote("FIXME: This is broken SECCOMP Failues?")
 
-		dockerfile := `FROM fedora-minimal
+		dockerfile := `FROM registry.fedoraproject.org/fedora-minimal
```
Comment (TomSweeneyRedHat) [inline test/e2e/exec_test.go:289]: I've run into troubles with this repo in the past, will keep my fingers crossed.

**Comment by TomSweeneyRedHat** [comment]
LGTM

File: test/e2e/common_test.go [hunk 2]
```diff
@@ -629,23 +631,33 @@ func isRootless() bool {
...
-		panic("SkipIfCgroupV2 must specify a reason to skip")
-	}
+	checkReason(reason)
 	if CGROUPSV2 {
 		Skip(reason)
 	}
 }
 
+func isContainerized() bool {
+	if os.Getenv("container") == "podman" {
```
Comment (nalind) [inline test/e2e/common_test.go:648]: Is this the only value of "container" for which the test calling this function should be skipped?

**Comment by ashley-cui** [comment]
LGTM

File: test/e2e/common_test.go (hunk 2 above)
Comment (rhatdan) [reply test/e2e/common_test.go:648]: Well we are running within Podman, but making this just check `if os.Getenv('container") != "" ` is probably better.

File: test/e2e/exec_test.go (hunk 1 above)
Comment (cevich) [reply test/e2e/exec_test.go:289]: This is (hopefully) only temporary until Brent can get local caching working.

File: test/e2e/common_test.go (hunk 2 above)
Comment (cevich) [reply test/e2e/common_test.go:648]: I believe so.  Though perhaps a `os.Getenv("container") != ""` would be safer.  I'll fix it, thanks.

**Review by vrothberg — COMMENTED** [review]
LGTM

**Pull request merged**